pages/user guides/bulk_edits.md

```diff
@@ -0,0 +1,132 @@
+
+{% include image.html file="edit-apps-4.png" %}
+
+## Bulk Assigning Server to Applications
+
+When working with servers, sometimes it's necessary to associate many of your existing servers to an application record. For this example, we're going to associate all servers with the fully qualified domain name (FQDN) 'Abcya.com' with the application named 'ABCya!'.
+
+First, use the advanced filtering drawer to filter your servers by FQDN.
+
+{% include image.html file="assign-servers-1.png" %}
+<br>
+
+Now, select the servers you'd like to assign.
+
+{% include image.html file="assign-servers-2.png" %}
+<br>
+
+Next, under the 'Bulk Actions' menu, click 'Assign to Application'.
+
+{% include image.html file="assign-servers-3.png" %}
+<br>
+
+Now select the application that you'd like to assign your selected servers to. For this example we will select 'ABCya!'. Once you've selected the app, click 'Assign'. 
+
+{% include image.html file="assign-servers-4.png" %}
+<br>
+
+The servers will now be assigned to your selected application.
+
+{% include image.html file="assign-servers-5.png" %}
+<br>
+
+## Copy Values Across Applications
+
+In Tidal Accelerator, it's considered best practice to model the same application in different environments as separate application records. This is because separate 'versions' of an application in different environments can have different needs when it comes to planning a migration. For example, an application running in a development environment has different uptime requirements, resources, and dependencies than that same application running in your production environment.
+
+However, these separate application records also share many of the same attributes, and it can be time consuming to enter the same data for each application record.
+
+To this end, you can copy values from an application to other versions of that application. We determine the 'other versions' as application records with the same name but different environments.
+
+For our example, say we have an application 'ABCya!', with all of it's relevant data captured. We also have two other version of this application in different environments that we want to copy some of this data to.
+
+First, from the application record page, click 'Copy To'.
+
+{% include image.html file="copy-apps-1.png" %}
+<br>
+
+Next you will be shown a list of applications to copy the values to. Select the application records and click 'Continue'.
+
+{% include image.html file="copy-apps-2.png" %}
+<br>
+
+Now you will be shown a list of the application values to copy over. These are presented alongside the existing values of your selected target applications, if present. Select any of the values that you'd like to copy over and click 'Continue'.
+
+{% include image.html file="copy-apps-3.png" %}
+<br>
+
+You will be shown a confirmation message explaining what you're about to do, and how many records it will affect. If you are satisfied, click 'Confirm'.
```

Comment:
If you're giving them an example you should be proactive. `You will be shown the confirmation message... . For our example x, y, and z are going to change. We're satisfied with this change, so click 'Confirm'.

Reply:
You switch from proactive to suggestive and back again throughout the guide. Which, from a guide perspective feels weird? But maybe that's just me 🤷

Reply:
Sorry not sure I follow - for each of the four sections here they're written like 'do this, then do this, etc', which I assume is what you mean by proactive. I'm suggestive in cases like 'you can use tags to filter your results' etc that aren't part of the core flow. I don't want to add that as a distinct step because I don't want to user to mistakenly assume that they must filter by tags to proceed, etc

Reply:
I don't think I want to write out the exact confirmation message that they'll get - they might be following along with their specific environment rather than using demo data and doing exactly what I'm doing. As long as the steps to follow are correct I feel as though that's sufficient.

Reply:
"If you are satisfied, click 'Confirm'" can be simplified to "and then click confirm" because that's what's happening in the demo images.

Reply:
I'm fully prepared to be wrong tho, let's see if we get any user feedback